static/javascript/index.js

```diff
@@ -33,6 +33,68 @@ function addNewNoteToDB(event){
 
     $.post("/add_note.json", formInputs, upDateNotes);
 }  
+
+////// Edit Exhisting Note //////
+function editExhistingNote(event){
+    event.preventDefault();
+
+    var note_id = $(this).val();
+
+    var note_title = '#note_title_' + note_id;
+
+    var notes_from_db = '#notes_from_db_' + note_id;
+    
+    $(note_title).attr("contenteditable", "true");
+    $(notes_from_db).attr("contenteditable", "true");
+
+
+    // Check if button has class 'edit_button' then toggle back to 'save_edits'
+    if ($(this).hasClass('edit_button')){
+        $(this).html('Save Edits').toggleClass('edit_button save_edits');
+    }
+    
+
+}
+
+////// Update DB with Edited Notes Content ////
+function editAndUpDateNotes(response){
+    console.log("editAndUpdateNotes running");
+}
+
+
+function updateDBwithEditedNote(event){
+    event.preventDefault();
+
+    var note_id = $(this).val();
+
+    var title_id = "#note_title_"+ note_id;
+    var note_content_id = "#notes_from_db_" + note_id;
+    
+    var note_title = $(title_id).html();
+    console.log(note_title);
+    var note_content = $(note_content_id).html();
+
+
+
+    // Check if button has class 'save_edits' then toggle back to 'edit_note'
+    if ($(this).hasClass('save_edits')){
+        $(this).html('Edit Note').toggleClass('save_edits edit_button');
+    }
+
+    
+    //TODO fix path to 
+    $.ajax({
+        url: "/notes/edit/"+ note_id,
+        type: 'POST',
+        data: {
+        "note_id" : note_id,
+        "note_title": note_title,
+        "note_content": note_content
```

Comment:
Send the note to the server using the same format that you receive the note from the server (the keys shouldn't have "note_" prefixes).